src/helpers/group/group-helper.js

```diff
@@ -4,7 +4,7 @@ const groupApi = getGroupApi();
 
 export async function fetchGroups({ limit, offset, name, orderBy }) {
   const [ groups, auth ] = await Promise.all([
-    groupApi.listGroups(limit, offset, name, undefined, undefined, orderBy),
+    groupApi.listGroups(limit, offset, name, undefined, undefined, undefined, undefined, undefined, orderBy),
```

Comment:
:scream: I think we might want to stop this argument madness and create some query builder or custom generator which will use an object for params instead. This is getting silly

(nothing you can do about it at the moment, just pointing out)

Reply:
@Hyperkid123 agree, I've wrote something for inventory but we'd need some generic way how to map object to params.